CODE_GENERATION.md

```diff
@@ -201,10 +201,9 @@ func preDelete(_ context.Context, cr *svcapitypes.Stage, obj *svcsdk.DeleteStage
 }
 ```
 
-If the external-name is decided by AWS after the creation, like in VPC, then you
-need to inject `postCreate` and make sure to `meta.SetExternalName(cr, id)` there
-and return `managed.ExternalCreation{ExternalNameAssigned: true}`.
-
+If the external-name is decided by AWS after the creation, then you
+need to inject `postCreate` to set the crossplane resource external-name to 
+the unique identifier of the resource, for eg see [`apigatewayv2`](https://github.com/crossplane/provider-aws/blob/master/pkg/controller/apigatewayv2/api/setup.go#L77)
```

Comment:
```suggestion
the unique identifier of the resource, for eg see [`apigatewayv2`](https://github.com/crossplane/provider-aws/blob/7f29cf3/pkg/controller/apigatewayv2/api/setup.go#L77)
```

We need to pin to the current commit because that line `master` might change in the future.